validator/delphi_validator/static.py

```diff
@@ -0,0 +1,399 @@
+"""Static file checks."""
+from os.path import join
+import re
+from datetime import date, datetime, timedelta
+import pandas as pd
+from .datafetcher import FILENAME_REGEX
+from .errors import ValidationError
+from .utils import GEO_REGEX_DICT
+
+class StaticValidation:
```

Comment:
Should these be named Static/DynamicValidator (instead of Validation)?

Reply:
Yes I think you may have seen a slightly earlier version

Reply:
i.e. they have already been changed

Reply:
ah sorry about that. I'm going through individual commits and commenting as I go.